docker-compose-java-development.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.5'
+services:
+
+  java-development:
+    image: eventuateio/eventuate-tram-sagas-examples-customers-and-orders-java-development:latest
+    volumes:
+      - .:/eventuate-tram-sagas-examples-customers-and-orders-java-development
+      - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysql/eventuate
```

Comment:
Are any of these environment variables needed?

Reply:
It seems no